projects/js-toolkit/packages/liferay-cli/src/dependencies.json

```diff
@@ -35,6 +35,13 @@
 		},
 		"devDependencies": {}
 	},
+	"facet-remote-app-react": {
+		"dependencies": {
+			"react": "^17.0.0",
+			"react-dom": "^17.0.0"
```

Comment:
Do we need to align these versions with the exact versions in liferay-portal?

Reply:
I had the same doubt :sweat_smile: , but then realized that we don't. This is because these are only used when you don't want to import the ones from Portal, so we can default to the latest ones in that case, maybe.

When you don't use the `platform-agnostic` target platform, you get `react` from the target platforms itself, transitively, so we don't need to put anything in the project's `package.json` file. :tada: